src/main/kotlin/ar/edu/unahur/obj2/semillasAlViento/Parcela.kt

```diff
@@ -1,45 +1,40 @@
 package ar.edu.unahur.obj2.semillasAlViento
 
-class Parcela(val ancho: Int, val largo: Int, val horasSolPorDia: Int) {
+class Parcela(private val ancho: Int, private val largo: Int, private val horasSolPorDia: Int) {
   val plantas = mutableListOf<Planta>()
-  var cantidadPlantas = 0
```

Comment:
En vez de borrarla directamente la podrían haber convertido en un método: 
```suggestion
  fun cantidadPlantas() = plantas.size
```